asynchronous-authorization/vercel-ai-node-js/README.md

```diff
@@ -43,13 +43,25 @@ OPENAI_API_KEY="openai-api-key"
 1. Install dependencies.
 
    ```sh
-   npm install
+   bun install
```

Comment:
I think we should keep npm install as thats more widely used and available by default